mods/ra/maps/soviet-06b/soviet06b-AI.lua

```diff
@@ -0,0 +1,112 @@
+IdleHunt = function(unit) if not unit.IsDead then Trigger.OnIdle(unit, unit.Hunt) end end
+
+IdlingUnits = function()
+	local lazyUnits = enemy.GetGroundAttackers()
+
+	Utils.Do(lazyUnits, function(unit)
+		Trigger.OnDamaged(unit, function()
+			Trigger.ClearAll(unit)
+			Trigger.AfterDelay(0, function() IdleHunt(unit) end)
+		end)
+	end)
+end
+
+BaseBuildings =
+{
+	{ "apwr", CVec.New(5, -9), 500, true },
+	{ "tent", CVec.New(-4, -4), 400, true },
+	{ "proc", CVec.New(0, -8), 1400, true },
+	{ "weap", CVec.New(-4, -8), 2000, true },
+	{ "apwr", CVec.New(6, -5), 500, true }
+}
+
+BuildBase = function()
+	if CYard.IsDead or CYard.Owner ~= enemy then
+		return
+	elseif Harvester.IsDead and enemy.Resources <= 299 then
+		return
+	end
+
+	for i,v in ipairs(BaseBuildings) do
+		if not v[4] then
```

Comment:
He is refering to the boolean value in the table (see 4th index). Are you sure that `v.exist` is correct then?